configuration/editors.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: documentation
+title: Editors
+---
+
+{% include base.html %}
+
+# Editors - Different ways to power up your textual configuration
+
+There are several existing solutions, that can help you configuring your openHAB instance in a textual way.
+Mainly there are two bigger projects for this. 
```

Comment:
I will change the sentence above too.
Otherwise we would have 2 sentences beginning with "there" which sounds bad while reading.